docs/source/model_design/risk_factors.rst

```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
 Population Attributable Fraction (PAF)
 ++++++++++++++++++++++++++++++++++++++
 
-The population attributable fraction is a quantification of the proportion of 
-of a given cause outcome, such as cases, deaths, or DALYs, that could be 
-eliminated by removing a risk exposure.
+Measures of population impact estimate the expected impact on a population of removing or changing the distribution of risk factors in that population. In GBD, this is the level of exposure that is equal to the TMREL. They take into account the both the strength of the association (estimated by a measure of effect, like the rate ratio) **and** the distribution of the risk factor in the population. It assumes that we have established that the association between disease and risk factor is *causal*. If this assumption is true, population impact estimates measure how much of the disease in the population is caused by the suspected risk factor. 
+
+Intuitively, PAF equals (O − E)/O, where O and E refer to the observed number of cases and the expected number of cases under no exposure, respectively. The term “attributable” has a causal interpretation: PAF is the estimated fraction of all cases that would not have occurred if there had been no exposure. As an example, in early 1950, Doll derived O = 11189 and E= 1875 using the Doll and Hill case-control study of smoking and lung cancer deaths throughout England and Wales, so the smoking PAF for lung cancer deaths was (11189 − 1875)/11189= 83%.
```

Comment:
define PAF before using acronym. "E" should be the number expected for TMRED, which is not always no exposure.  Perhaps lead with the Doll example, and then generalize.

Reply:
ok!